### DISCUSSION TITLE
How to remove wineskin from Mac?

### DISCUSSION THREAD

**Discussion by nysni** [The-Wineskin-Project/WineskinServer#535] [Q&A] [answered]
I already ran this in homebrew: % brew install --cask --no-quarantine gcenx/wine/wineskin
Already downloaded and gstreamer was downloaded as well.
How can I remove it? I tried uninstall and amongst other things... but it is still not removed..

**Comment by Gcenx** [comment, marked as answer]
Remove
> ~/Library/Application Support/Wineskin

Any generated wrappers you need to trash manually as those are self contained app bundles.

You can uninstall GStreamer using brew
